### PR TITLE
feat: clean btc lps into wbtc and topup gas station

### PR DESCRIPTION
**Pull request by petrovska-petro**
Tackles #1045 & #884 conditionally

```
brownie run treasury/wbtc_fee_collector main 5000000000000000000
```

### CODE REVIEW

File: scripts/treasury/wbtc_fee_collector.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        min_out_wbtc_from_im_btc,
+        wbtc,
+        trops,
+        r.treasury_tokens.mBTC,
+        True,
+    )
+
+    # 3. Top-up gas station
+    # NOTE: not added the withdraw from `weth` since it breaks the flow of the scripts by reverting
+    if top_eth_mantissa > 0:
```
Comment (gosuto-inzasheru) [inline scripts/treasury/wbtc_fee_collector.py:84]: think this needs a cast? `Decimal` works well on the cli arg strings

File: scripts/treasury/wbtc_fee_collector.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        True,
+        min_out_wbtc_from_im_btc,
+        wbtc,
+        trops,
+        r.treasury_tokens.mBTC,
+        True,
+    )
+
+    # 3. Top-up gas station
+    # NOTE: not added the withdraw from `weth` since it breaks the flow of the scripts by reverting
```
Comment (gosuto-inzasheru) [inline scripts/treasury/wbtc_fee_collector.py:83]: why doest weth.withdraw revert..?

Comment (gosuto-inzasheru) [reply scripts/treasury/wbtc_fee_collector.py:83]: ah it is that thing due to some weird eth fallback behaviour in the gnosis safe contract.

sure i guess we can skip for now without having to build calldata manually...

**Review by gosuto-inzasheru — CHANGES_REQUESTED** [review]

File: scripts/treasury/wbtc_fee_collector.py (hunk 1 above)
Comment (petrovska-petro) [reply scripts/treasury/wbtc_fee_collector.py]: good catch, pushed in 5c3a8483fa67b7a8cc0fc7246d30f445fe5ca19a

File: scripts/treasury/wbtc_fee_collector.py (hunk 2 above)
Comment (petrovska-petro) [reply scripts/treasury/wbtc_fee_collector.py:84]: exactly that issue with the gnosis contract 

```
receive() external payable {
        emit SafeReceived(msg.sender, msg.value);
    }
```

**Review by gosuto-inzasheru — APPROVED** [review]

**Review by jalbrekt85 — APPROVED** [review]
nice, lgtm

**Pull request merged**